car_manager.py

```diff
@@ -0,0 +1,38 @@
+import random
+from turtle import Turtle
+COLORS = ["red", "orange", "yellow", "green", "blue", "purple"]
```

Comment:
Very good that you used all caps for constants.

Reply:
Thanks